sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTraceExporter.cs

```diff
@@ -40,7 +42,7 @@ public override ExportResult Export(in Batch<Activity> batch)
 
             try
             {
-                var telemetryItems = TraceHelper.OtelToAzureMonitorTrace(batch, TraceResource, _instrumentationKey);
+                var telemetryItems = TraceHelper.OtelToAzureMonitorTrace(batch, TraceResource, _instrumentationKey, _sampleRate);
```

Comment:
`_sampleRate` is a constant value, we should avoid passing that as a parameter every time. Shall we refactor it?
We could take the learning for instrumentation key later.

Reply:
You could also consider in a different PR.

Reply:
I was thinking about that, three of those parameters never change.

It is possible that a user could have two TraceExporters, each with different sampleRates. In this case, we can't store the sampleRate value in the TraceHelper.
But we could change the TraceHelper to be instance based instead of a static class and store these constant parameters within the instance.

This would be a large refactor and I think out of scope for this PR, but let me know if you have any other ideas :)